src/Talis/Persona/Client/Base.php

```diff
@@ -93,6 +93,10 @@ public function __construct(array $config)
         }
 
         $this->logger = $this->get($config, 'logger', null);
+/* echo "\n\n== data fetch =========\n\n"; */
+/* var_dump($config); */
+/* echo "\n\n===========\n\n"; */
+/* die(); */
```

Comment:
:eyes:

Reply:
Yeah - This is for an early 30% review of the approach.

I'm still working on getting the tests fixed. It will need tidying up etc afterwards.

It's only really a review of the approach I'm looking for.

Reply:
:see_no_evil: